boards/nucleo-common/include/board_common.h

```diff
@@ -41,16 +41,18 @@ extern "C" {
  * @{
  */
 #ifdef CPU_MODEL_STM32F302R8
+#define LED0_GPIO           GPIOB
```

Comment:
No this is different from port: here it's PORT_B. This macro targets the GPIO port address directly.
I made the same confusion when working on #6991.